src/hotspot/share/opto/reg_split.cpp

```diff
@@ -272,15 +273,15 @@ uint PhaseChaitin::split_USE(MachSpillCopyNode::SpillType spill_type, Node *def,
   }
 
   Node *spill = get_spillcopy_wide(spill_type, def, use, useidx );
-  if( !spill ) return 0;        // Bailed out
+  if( !spill ) return -1;        // Bailed out
   // Insert SpillCopy before the USE, which uses the reaching DEF as
   // its input, and defs a new live range, which is used by this node.
   insert_proj( b, bindex, spill, maxlrg++ );
   // Use the spill/clone
   use->set_req(useidx,spill);
 
-  // return updated live range count
-  return maxlrg;
+  // return generated node count
```

Comment:
The comment is confusing. I would suggest to add comment at the header of this method to describe all returned values [-1, 0, 1].

Reply:
Add comment in method header of split_USE and remove this comment.